infra/scripts/test_android_runtime_release.sh

```diff
@@ -0,0 +1,140 @@
+SKIPLIST_INTERP=$(grep -v '#' "${ROOT_PATH}/Product/aarch64-android.release/out/unittest/nnapi_gtest.skip.noarch.interp" | tr '\n' ':')
+$ADB_CMD shell LD_LIBRARY_PATH=$ANDROID_WORKDIR/Product/lib EXECUTOR=Interpreter BACKENDS=$BACKEND $ANDROID_WORKDIR/Product/unittest/nnapi_gtest \
+                                                            --gtest_output=xml:$ANDROID_REPORT_DIR/nnapi_gtest_interp_Interpreter.xml \
+                                                            --gtest_filter=-$SKIPLIST_INTERP
+$ADB_CMD shell LD_LIBRARY_PATH=$ANDROID_WORKDIR/Product/lib EXECUTOR=Interpreter BACKENDS="" DISABLE_COMPILE=1 sh $ANDROID_WORKDIR/tests/scripts/models/run_test_android.sh \
+                                                        --driverbin=$ANDROID_WORKDIR/Product/bin/nnapi_test \
+                                                        --reportdir=$ANDROID_REPORT_DIR \
+                                                        --tapname=nnapi_test_interp_Interpreter.tap ${MODELLIST_INTERP:-}
+
+# Testing Mixed Backend
+MODELLIST_UNION=$(cat "${UNION_MODELLIST_PREFIX}.intersect.txt")
+SKIPLIST_UNION=$(grep -v '#' "$UNION_SKIPLIST_PREFIX.union" | tr '\n' ':')
+$ADB_CMD shell LD_LIBRARY_PATH=$ANDROID_WORKDIR/Product/lib OP_BACKEND_Conv2D="cpu" OP_BACKEND_MaxPool2D="acl_cl" OP_BACKEND_AvgPool2D="acl_neon" ACL_LAYOUT="NCHW" BACKENDS="acl_cl\;acl_neon\;cpu" $ANDROID_WORKDIR/Product/unittest/nnapi_gtest \
+                                                            --gtest_output=xml:$ANDROID_REPORT_DIR/nnapi_gtest_mixed.xml \
+                                                            --gtest_filter=-${SKIPLIST_UNION}
+$ADB_CMD shell LD_LIBRARY_PATH=$ANDROID_WORKDIR/Product/lib OP_BACKEND_Conv2D="cpu" OP_BACKEND_MaxPool2D="acl_cl" OP_BACKEND_AvgPool2D="acl_neon" ACL_LAYOUT="NCHW" BACKENDS="acl_cl\;acl_neon\;cpu" sh $ANDROID_WORKDIR/tests/scripts/models/run_test_android.sh \
+                                                        --driverbin=$ANDROID_WORKDIR/Product/bin/nnapi_test \
+                                                        --reportdir=$ANDROID_REPORT_DIR \
+                                                        --tapname=nnapi_test_mixed.tap ${MODELLIST_UNION:-}
+
+# Testing Unittest_standalone
+for TEST_BIN in `find ${ROOT_PATH}/Product/aarch64-android.release/out/unittest_standalone -maxdepth 1 -type f -executable`; do
+  $ADB_CMD shell "cd $ANDROID_WORKDIR/Product/unittest_standalone;
+                  LD_LIBRARY_PATH=$ANDROID_WORKDIR/Product/lib ./$(basename $TEST_BIN) \
+                  --gtest_output=xml:$ANDROID_REPORT_DIR/$(basename $TEST_BIN).xml"
+done
+
+# This is test for profiling.
+# $ADB_CMD shell mkdir -p $ANDROID_REPORT_DIR/benchmark
+# $ADB_CMD shell 'cd $ANDROID_WORKDIR && LD_LIBRARY_PATH=$ANDROID_WORKDIR/Product/lib sh $ANDROID_WORKDIR/tests/scripts/test_scheduler_with_profiling_android.sh'
```

Comment:
This comment is intended?

Reply:
Oh, That's a some test for initial code. I will remove it.